wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsworkunits/WUCreateAndUpdateWrapper.java

```diff
@@ -1,711 +1,723 @@
+        result.setWuid(wuid);
+        result.setXmlParams(xmlParams);
+        if (this.getApplicationValues().size() > 0)
+        {
+            org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ApplicationValue[] appvals = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ApplicationValue[this.getApplicationValues().size()];
+            for (int i = 0; i < this.getApplicationValues().size(); i++)
+            {
+                org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ApplicationValue av = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ApplicationValue();
+                av.setApplication(this.getApplicationValues().get(i).getApplication());
+                av.setName(this.getApplicationValues().get(i).getName());
+                av.setValue(this.getApplicationValues().get(i).getValue());
+                appvals[i] = av;
+            }
+            org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ArrayOfApplicationValue appvalsarray = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ArrayOfApplicationValue();
+            appvalsarray.setApplicationValue(appvals);
+                    
+            result.setApplicationValues(appvalsarray);
+        }
+
+        if (this.getDebugValues().size() > 0)
+        {
+            org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.DebugValue[] debugvals = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.DebugValue[this.getDebugValues().size()];
+            for (int i = 0; i < this.getDebugValues().size(); i++)
+            {
+                org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.DebugValue av = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.DebugValue();
+                av.setName(this.getDebugValues().get(i).getName());
+                av.setValue(this.getDebugValues().get(i).getValue());
+                debugvals[i] = av;
+            }
+            org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ArrayOfDebugValue debugvalsarray = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_62.ArrayOfDebugValue();
+            debugvalsarray.setDebugValue(debugvals);
+            result.setDebugValues(debugvalsarray);
+        }
+        return result;
+    }
+
+    public org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_69.WUCreateAndUpdate getRawVersion1_69()
+    {
+        org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_69.WUCreateAndUpdate result = new org.hpccsystems.ws.client.gen.axis2.wsworkunits.v1_69.WUCreateAndUpdate();
+
+        result.setProtected(_protected);
+        result.setAction(action);
+        result.setAddDrilldownFields(addDrilldownFields);
+        result.setClusterOrig(clusterOrig);
+        result.setClusterSelection(clusterSelection);
+        result.setDescription(description);
+        result.setDescriptionOrig(descriptionOrig);
+        result.setJobname(jobname);
+        result.setJobnameOrig(jobnameOrig);
+        if (priorityClass != null) 
```

Comment:
Should we test for action != null before line 435, or is action required?

Reply:
Action is a string, so null is an acceptable value. Only ints/booleans would throw NPEs.

Reply:
Right, I mentioned it because it's declared as:
private Integer                       action;

Reply:
I wrapped the remaining Integer wrapper fields in null checks in this method.